src/libcollections/btree/node.rs

```diff
@@ -43,17 +53,39 @@ use boxed::Box;
 const B: usize = 6;
 pub const CAPACITY: usize = 2 * B - 1;
 
+/// The underlying representation of leaf nodes. Note that is often unsafe to actually store
+/// these, since only the first `len` keys and values are assumed to initialized. See also
```

Comment:
That clarification seems reasonable. What do you mean by "to be uninit"? The first `len` keys and values are initiailized.